frontend/packages/design-system/src/assets/icons/CircleError.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Path, Circle, G } from 'react-native-svg';
+
+import { tokens } from '../../index';
+
+export const CircleError = () => (
+  <G>
+    <Circle cx="10" cy="10" r="10" fill={tokens.colors.signalDanger} />
```

Comment:
We shouldn't set the fill in the components. This should be `currentColor` so we can customise it when using the icon